src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayViewController.swift

```diff
@@ -34,19 +34,7 @@ class DiaryDayViewController: UIViewController, UITableViewDataSource, UITableVi
 
 		view.backgroundColor = .enaColor(for: .darkBackground)
 
-		/*
-		    In iOS 15, the tab bar background automatically adjusts it self, when there is no content on the back it becomes transparent
-		    where as it has a background when there is content on the back. Unfortunately, in our case the tab bar remains transparent even
-		    through there is content on the back, so we fix this by overriding the appearance with a background.
-			Solution is inspired from: https://developer.apple.com/forums/thread/682420
-		*/
-		if #available(iOS 15, *) {
-			let appearance = UITabBarAppearance()
-			appearance.configureWithOpaqueBackground()
-			appearance.backgroundColor = .enaColor(for: .backgroundLightGray)
-			tabBarController?.tabBar.standardAppearance = appearance
-			tabBarController?.tabBar.scrollEdgeAppearance = tabBarController?.tabBar.standardAppearance
-		}
+		setupTabBarAppearance()
```

Comment:
We don't need this here, if we have it in viewWillAppear

Reply:
Thanks. I have removed it from viewDidLoad and keep it just in viewWillAppear.